mrp_bom_component_cost/models/mrp_bom_line.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MrpBomLine(models.Model):
+    _inherit = "mrp.bom.line"
+
+    currency_id = fields.Many2one(related="bom_id.currency_id")
+    cost_unit = fields.Float(string="Unit Cost", compute="_compute_cost_unit")
+    cost_subtotal = fields.Float(
+        string="Subtotal Cost", compute="_compute_cost_subtotal"
+    )
+
+    @api.depends("product_id", "product_uom_id", "currency_id")
+    def _compute_cost_unit(self):
+        for rec in self:
+            if rec.product_id:
+                cost_unit = product_cost_unit = rec.product_id.standard_price
```

Comment:
It is taken into account that this cost is indicative, since it is not the actual cost.

Reply:
I have not taken that into account. What do you mean?

Reply:
@Jaimermaccione @manuelregidor Can you explain to @Tisho99  the different types of costs?

Reply:
@ValentinVinagre Sorry, but I don't get what you mean.